moqt-server/src/modules/track_namespace_manager.rs

```diff
@@ -200,13 +222,37 @@ impl TrackNamespaces {
             bail!("subscriber not found");
         }
         track_name_object.delete_subscriber(subscriber_session_id);
+
+        // Delete track with no subscribers
```

Comment:
Sorry, it isn't an issue specific to this PR, But I think that track should not be deleted when track has no subscribers.

Having no subscribers doesn't necessarily mean that the track has completed publishing.

Reply:
Publishers cannot ANNOUNCE to server which track it has.
Currently, who requests the track are subscribers, so if there is no subscriber for the track, the server has to remove the track from its management.

This process is not added in this PR, but implemented based on the sequence diagram in #55 .

Reply:
We agreed that this implementation is correct.

[Note]
- ANNOUNCE message contains TrackNameSpace but not TrackName.
- Only SUBSCRIBE_OK message contains TrackName.
- Track is managed in Publisher only, so that we can delete TrackName in Server.